Source/Scene/Model.js

```diff
@@ -5317,7 +5330,9 @@ Model.prototype.update = function (frameState) {
         loadResources.resourcesParsed &&
         loadResources.pendingShaderLoads === 0
       ) {
-        ModelOutlineLoader.outlinePrimitives(this);
+        if (this.showOutline) {
+          ModelOutlineLoader.outlinePrimitives(this);
+        }
```

Comment:
Could you add a test for this in `ModelSpec.js`? I think it can be as simple as loading any model with `showOutline: false` and checking that `ModelOutlineLoader.outlinePrimitives` is not called (using a spy).